lib/runtime2/threepio/capabilities/generate-domcument/TaskDecomposer.ts

```diff
@@ -0,0 +1,22 @@
+import { FragmentType, ParsedModule, MoudleFragmentTask } from './interfaces';
+
+/**
+ * Converts a single parsed module into a SkeletonNode and an FragmentTask.
+ * This is designed to be called incrementally as modules are parsed.
+ * @param module Parsed module information.
+ * @param overallDesignTheme The overall design theme from the page header.
+ * @param appName The application name from the page header.
+ * @returns An object containing the new SkeletonNode and its corresponding FragmentTask.
+ */
+export function createModuleTask(module: ParsedModule, overallDesignTheme: string, parentId: string): { task: MoudleFragmentTask } {
```

Comment:
```suggestion
export function createModuleTask(module: ParsedModule, overallDesignTheme: string, parentId: string): MoudleFragmentTask {
```
是不是可以直接返回 `MoudleFragmentTask`